nepi/templates/dashboard/student_details.html

```diff
@@ -7,7 +7,7 @@
 {% block content %}
     <ul class="breadcrumb">
       <li><a href="/dashboard/#user-groups">Dashboard</a> <span class="divider">/</span></li>
-      <li><a href="/group_details/{{group.id}}/">{{group.name}}</a> <span class="divider">/</span></li>
+      <li><a href="/roster_details/{{group.id}}/">Roster</a> <span class="divider">/</span></li>
```

Comment:
why not `{% url 'roster-details' group.id %}` here since you named it in urls.py?